### PR TITLE
Make pagination optional for API list notification endpoints

### PR DESCRIPTION
**Pull request by 6543**
make pagination optional for list notification endpoints

### CODE REVIEW

File: routers/api/v1/utils/utils.go [hunk 1]
```diff
@@ -41,8 +41,12 @@ func GetQueryBeforeSince(ctx *context.APIContext) (before, since int64, err erro
 
 // GetListOptions returns list options using the page and limit parameters
 func GetListOptions(ctx *context.APIContext) models.ListOptions {
+	page := ctx.QueryInt("page")
+	if page == 0 {
+		return models.ListOptions{}
```
Comment (6543) [inline routers/api/v1/utils/utils.go]: Not sure if this is required ... will test ...

**Comment by 6543** [comment]
@techknowlogick would be nice if you can add this to 1.12 :)

**Review by lunny — APPROVED** [review]

**Review by adelowo — APPROVED** [review]

**Comment by techknowlogick** [comment]
I can't update branch on this PR, please mark as allow edits from maintainers :)

**Comment by zeripath** [comment]
make lg-tm work

**Pull request merged**